LegendaryExplorer/LegendaryExplorer/Tools/ConditionalsEditor/ConditionalsEditorWindow.xaml.cs

```diff
@@ -172,7 +172,8 @@ private void Search()
                 try
                 {
                     string text = entry.Conditional.Decompile();
-                    if (text.Contains(searchText, StringComparison.OrdinalIgnoreCase))
+                    string entryPlotPath = entry.PlotPath;
```

Comment:
`PlotPath` will be null for any conditional not in the plot database (such as ones from mods), causing an exception to be thrown on the next line.
```suggestion
                    string entryPlotPath = entry.PlotPath ?? "";
```